denops/ddu/ddu.ts

```diff
@@ -848,7 +870,9 @@ export class Ddu {
   quit() {
     // NOTE: quitted flag must be called after ui.quit().
     this.#quitted = true;
-    this.#aborter.abort({ reason: "quit" });
+    const reason = new QuitAbortReason();
+    this.#aborter.abort(reason);
+    /* no await */ this.#cancelGatherStates([], reason);
```

Comment:
This is broken.

Reply:
It must not be canceled.  Because if it is canceled, resume feature does not work.